apis/sgv2-quarkus-common/CONFIGURATION.md

```diff
@@ -46,13 +46,15 @@
-| `stargate.metrics.tenant-request-counter.metric-name` | `String`             | `http.server.requests.counter`      | Name of the metric.                                                     |
-| `stargate.metrics.tenant-request-counter.tenant-tag`  | `String`             | `tenant`                            | The tag key for tenant id.                                              |
-| `stargate.metrics.tenant-request-counter.error-tag`   | `String`             | `error`                             | The tag key for the request error flag (true/false).                    |
+| Property                                                         | Type                 | Default                             | Description                                                                                                                                                  |
+|------------------------------------------------------------------|----------------------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `stargate.metrics.global-tags`                                   | `Map<String,String>` | `{"module": "docsapi"}`             | Map of global tags that will be applied to every meter.                                                                                                      |
+| `stargate.metrics.tenant-request-counter.enabled`                | `boolean`            | `${stargate.multi-tenancy.enabled}` | If extra metric for counting the request per tenant should be recorded.                                                                                      |
+| `stargate.metrics.tenant-request-counter.metric-name`            | `String`             | `http.server.requests.counter`      | Name of the metric.                                                                                                                                          |
+| `stargate.metrics.tenant-request-counter.tenant-tag`             | `String`             | `tenant`                            | The tag key for tenant id.                                                                                                                                   |
+| `stargate.metrics.tenant-request-counter.error-tag`              | `String`             | `error`                             | The tag key for the request error flag (true/false).                                                                                                         |
+| `stargate.metrics.tenant-request-counter.user-agent-tag`         | `String`             | `agent`                             | The tag key for the user agent, if capturing is enabled.                                                                                                     |
```

Comment:
maybe `user-agent` as the default to be the most clear about what this is?

Reply:
Sure, I can adapt.

Reply:
@jeffreyscarpenter so usually tags names are defined as:

* `user_agent` - with underscore
* `userAgent` - camel-case

Would like to follow one of those two, any preference?

Reply:
I guess underscore